test/test_batch_consistency.py

```diff
@@ -83,6 +83,11 @@ def test_sliding_window_cmn(self):
         _test_batch(F.sliding_window_cmn, waveform, center=False, norm_vars=True)
         _test_batch(F.sliding_window_cmn, waveform, center=False, norm_vars=False)
 
+    def test_vad(self):
+        filepath = common_utils.get_asset_path("vad-hello-mono-32000.wav")
+        waveform, _ = torchaudio.load(filepath)
+        _test_batch(F.vad, waveform, sample_rate=32000)
```

Comment:
nit: is the `sample_rate=32000` different than what `waveform, sample_rate = torchaudio.load(...)` would return as `sample_rate`?